File: js/programmers/level_0/개미 군단.js
// Lv. 0 개미 군단

/**
 * @param {number} hp
 * @returns {number}
 */

const solution = (hp) => {
    return Math.floor(hp / 5) + Math.floor((hp % 5) / 3) + ((hp % 5) % 3);
};

File: java/programmers/level_0/src/개미_군단.java
public class 개미_군단 {

  public static int solution(int hp) {
    int answer = hp / 5;
    hp %= 5;

    answer += hp / 3;
    hp %= 3;

    answer += hp / 1;

    return answer;
  }

  public static void main(String[] args) {
    System.out.println(solution(242));
  }
}
